nercst/core/coords_converter.py

```diff
@@ -0,0 +1,28 @@
+import astropy.units as u
+import xarray as xr
+
+from neclib import config
+from astropy.time import Time
+from neclib.coordinates import parse_frame
+from astropy.coordinates import SkyCoord
+from neclib.parameters import PointingError
+
+
+def add_celestial_coords(array: xr.DataArray, frame: str) -> xr.DataArray:
+
+    pe = PointingError("OMU1P85M")
```

Comment:
Is it OK that it supports only 1.85m? I'm sorry I don't grasp its specification.
If you assume that at this time, it looks good to me.

Reply:
This issue is solved in #55 . The new one is to load the actual pointing parameter file simultaneously saved during the observation.